lexicons/tools/ozone/moderation/defs.json

```diff
@@ -210,6 +210,10 @@
         "durationInHours": {
           "type": "integer",
           "description": "Indicates how long the takedown should be in effect before automatically expiring."
+        },
+        "acknowledgeAllSubjectsOfAccount": {
+          "type": "boolean",
+          "description": "Set to true if all subjects authored by the account should be resolved."
```

Comment:
"If true, all other reports on content authored by this account will be resolved (acknowledged)"